diff --git a/node_modules/mathquill/build/mathquill.js b/node_modules/mathquill/build/mathquill.js
index a6f7939..5d05e69 100644
--- a/node_modules/mathquill/build/mathquill.js
+++ b/node_modules/mathquill/build/mathquill.js
@@ -8,7 +8,7 @@
  * one at http://mozilla.org/MPL/2.0/.
  */
 
-(function() {
+export default function mathquillFunc() {
 
 var jQuery = window.jQuery,
   undefined,
@@ -4979,4 +4979,4 @@ for (var key in MQ1) (function(key, val) {
   else MathQuill[key] = val;
 }(key, MQ1[key]));
 
-}());
+};
diff --git a/node_modules/mathquill/package.json b/node_modules/mathquill/package.json
index 52c23d8..1def280 100644
--- a/node_modules/mathquill/package.json
+++ b/node_modules/mathquill/package.json
@@ -1,7 +1,7 @@
 {
   "name": "mathquill",
   "description": "Easily type math in your webapp",
-  "version": "0.10.1a",
+  "version": "0.10.1-a",
   "license": "MPL-2.0",
   "repository": {
     "type": "git",
