diff --git a/node_modules/patch-package/dist/applyPatches.js b/node_modules/patch-package/dist/applyPatches.js
index 2d6344b..5dc09b5 100644
--- a/node_modules/patch-package/dist/applyPatches.js
+++ b/node_modules/patch-package/dist/applyPatches.js
@@ -42,7 +42,7 @@ function getInstalledPackageVersion({ appPath, path, pathSpecifier, isDevOnly, p
     }
     const { version } = require(path_2.join(packageDir, "package.json"));
     // normalize version for `npm ci`
-    const result = semver_1.default.valid(version);
+    const result = semver_1.default.valid(version, { loose: true });
     if (result === null) {
         throw new PatchApplicationError(`${chalk_1.default.red("Error:")} Version string '${version}' cannot be parsed from ${path_2.join(packageDir, "package.json")}`);
     }
